subprojects/core/src/main/java/org/gradle/api/internal/artifacts/JavaEcosystemAttributesDescriber.java

```diff
@@ -255,19 +255,14 @@ private static void describeLibraryElements(Object le, StringBuilder sb) {
         }
     }
 
-    @SuppressWarnings("deprecation")
     private static void describeUsage(Object usage, StringBuilder sb) {
         String str = toName(usage);
         switch (str) {
             case Usage.JAVA_API:
-            case Usage.JAVA_API_CLASSES:
-            case Usage.JAVA_API_JARS:
-                sb.append("compile-time");
+                sb.append("an API");
                 break;
             case Usage.JAVA_RUNTIME:
-            case Usage.JAVA_RUNTIME_CLASSES:
-            case Usage.JAVA_RUNTIME_JARS:
-                sb.append("runtime");
+                sb.append("a runtime");
                 break;
```

Comment:
I think these need to stick around too since they're explaining the usage even with deprecated usages.